packages/shared/lib/services/notification/webhook.service.ts

```diff
@@ -302,45 +302,58 @@ class WebhookService {
                     content: `Auth Webhook failed to send to ${webhookUrl}. The error was: ${errorMessage}`,
                     timestamp: Date.now()
                 });
-                await logCtx?.error(`Auth Webhook failed to send to ${webhookUrl}`, { error: e });
+                await logCtx?.error(`Auth Webhook failed to send to ${webhookUrl}`, { error: err });
             }
         }
     }
-    async forward(
-        environment_id: number,
-        providerConfigKey: string,
-        connectionIds: string[],
-        provider: string,
-        payload: Record<string, any> | null,
-        webhookOriginalHeaders: Record<string, string>,
-        logContextGetter: LogContextGetter
-    ) {
-        const { send, environmentInfo } = await this.shouldSendWebhook(environment_id, { forward: true });
+
+    async forward({
+        integration,
+        account,
+        connectionIds,
+        payload,
+        webhookOriginalHeaders,
+        logContextGetter
+    }: {
+        integration: Config;
```

Comment:
I replaced the floating values with `integration` easier to Type and provide more info